compiler/luci/pass/src/TransformMinMaxToRelu6Pass.test.cpp

```diff
@@ -0,0 +1,135 @@
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/TransformMinMaxToRelu6Pass.h"
+
+#include <luci/IR/CircleNodes.h>
+
+#include <gtest/gtest.h>
+
+namespace
+{
+
+/**
+ *  Minimum-Maximum pattern graph
+ *
+ *  [CircleInput]  [CircleConst]
+ *         \         /
+ *    [CircleMinimum]   [CircleConst]
+ *             |       /
+ *       [CircleMaximum]
+ *             |
+ *       [CircleOutput]
+ */
+class MinMaxGraph : public ::testing::Test
+{
+protected:
+  MinMaxGraph()
+  {
+    const int N = 1;
+    const int H = 4;
+    const int W = 4;
+    const int C = 3;
+
+    // graph input and output
+    auto graph_input = _g.inputs()->create();
+    auto graph_output = _g.outputs()->create();
+
+    // CircleInput
+    _input = _g.nodes()->create<luci::CircleInput>();
+    _input->index(graph_input->index());
+    _input->shape({N, H, W, C});
+    _input->dtype(loco::DataType::FLOAT32);
+
+    // CircleConst
+    _mini_const = _g.nodes()->create<luci::CircleConst>();
+    _mini_const->shape({}); // scalar
+    _mini_const->dtype(loco::DataType::FLOAT32);
+    _mini_const->size<loco::DataType::FLOAT32>(1);
+    _mini_const->at<loco::DataType::FLOAT32>(0) = 6.;
+
+    // CircleMinimum
+    _mini = _g.nodes()->create<luci::CircleMinimum>();
+    _mini->x(_input);
+    _mini->y(_mini_const);
+    _mini->shape({N, H, W, C});
+    _mini->dtype(loco::DataType::FLOAT32);
+
+    // CircleConst
+    _maxi_const = _g.nodes()->create<luci::CircleConst>();
+    _mini_const->shape({}); // scalar
+    _maxi_const->dtype(loco::DataType::FLOAT32);
+    _maxi_const->size<loco::DataType::FLOAT32>(1);
+    _maxi_const->at<loco::DataType::FLOAT32>(0) = 0.;
+
+    // CircleMaximum
+    _maxi = _g.nodes()->create<luci::CircleMaximum>();
+    _maxi->x(_mini);
+    _maxi->y(_maxi_const);
+    _maxi->shape({N, H, W, C});
+    _maxi->dtype(loco::DataType::FLOAT32);
+
+    // CircleOutput
+    _output = _g.nodes()->create<luci::CircleOutput>();
+    _output->index(graph_output->index());
+    _output->from(_maxi);
+    _output->shape({N, H, W, C});
+    _output->dtype(loco::DataType::FLOAT32);
+  }
+
+protected:
+  loco::Graph _g;
+  luci::CircleInput *_input = nullptr;
+  luci::CircleMinimum *_mini = nullptr;
+  luci::CircleConst *_mini_const = nullptr;
+  luci::CircleMaximum *_maxi = nullptr;
+  luci::CircleConst *_maxi_const = nullptr;
+  luci::CircleOutput *_output = nullptr;
+};
+
+} // namespace
+
+/**
+ *  Optimized graph looks like below.
+ *
+ *  [CircleInput]
+ *        |
+ *  [CircleRelu6]
+ *        |
+ *  [CircleOutput]
+ */
+TEST_F(MinMaxGraph, simple_test)
```

Comment:
@seanshpark @jinevening 
I followed the suggestion to change `TEST` to `TEST_F` yesterday. But on second thought, it seems natural to use just `TEST` I think. As [document](https://github.com/YOU-i-Labs/googletest/blob/master/googletest/docs/Primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests), `TEST_F` is used to test the fixture we want to test. For example, say if you want to test a `Queue` class, you should write a `QueueTest` having `Queue` as an its attribute and then call `TEST_F(QueueTest, ...)`. 

Likewise, if we want to test `Pass` struct with `TEST_F`, we should write a `PassTest` class, which BTW, looks unnatural because `Pass` structs is more like method(wrapping `run`) rather than a fixture. I mean, current `TEST_F(MinMaxGraph, ...)` or `TEST_F(FoldS64AddV2Test, ...)` looks like it want to test `Graph` rather than its pass.

So, just
```cxx
TEST(..., ...)
{
  MinMaxGraph g;
  luci::TransformMinMaxToRelu6Pass pass;
  auto ret = pass.run(&g._g);
  ...
}
```
is enough I think.

Please give me your opinions.

Reply:
As I know for this PR, you're adding `TransformMinMaxToRelu6Pass` and want to add a test for this class.
For me, `TransformMinMaxToRelu6PassTest` is natural for this point.

> Pass structs is more like method(wrapping run) rather than a fixture.

Pass has a simple structure with one method `run`.  I don't see any problem with this. 
You used word `fixture`, which I have difficulty understanding the meaning of this word in the sentence.